### ISSUE TITLE
Package/Recipe is out of date

### ISSUE THREAD

**Issue by wilderfield** [conda-forge/tvm-py-feedstock#10]
```$ python relay_quick_start.py
Traceback (most recent call last):
  File "/home/bryanloz/TVM/relay_quick_start.py", line 18, in <module>
    from tvm.contrib import graph_executor
ImportError: cannot import name 'graph_executor' from 'tvm.contrib' (/home/bryanloz/anaconda3/envs/tvm/lib/python3.9/site-packages/tvm/contrib/__init__.py)
```

https://github.com/apache/tvm/blob/main/python/tvm/contrib/graph_executor.py

**Comment by ngam** [comment]
@wilderfield are you interested in helping out here?

**Comment by ngam** [comment]
Should be resolved by #12
